bluemira/geometry/compound.py

```diff
@@ -59,18 +68,28 @@ def _create(cls, obj: cadapi.apiCompound, label="") -> BluemiraCompound:
         if not obj.isValid():
             raise GeometryError(f"Compound {obj} is not valid.")
 
-        bm_solids = [BluemiraSolid._create(solid) for solid in cadapi.solids(obj)]
-        bm_shells = [BluemiraShell._create(shell) for shell in cadapi.shells(obj)]
-        bm_faces = [BluemiraFace._create(face) for face in cadapi.faces(obj)]
-        bm_wires = [BluemiraWire(wire) for wire in cadapi.wires(obj)]
-        if len(bm_wires) == 0:
-            # In some edge cases there are edges that are not captured within wires
-            bm_wires = [
+        topo_compound_shapes = []
```

Comment:
This block implicitly ignores shapes that are below that level in the stack, do we need to either document or error if an object with multiple levels is input?
Implicitly at the moment it will get passed to the `_compound_obj` but will be ignored in the bluemira shapes.